src/main/connector/database/Database/Database.yaml

```diff
@@ -0,0 +1,217 @@
+metrics:
+  db.aborted.clients:
+    description: The number of connections that were aborted.
+    type: Counter
+    unit: "{connection}"
+  db.aborted.connects:
+    description: The number of failed attempts to connect to the MySQL server.
+    type: Counter
+    unit: "{connection}"
+  db.io.size{db.io.direction="received"}:
```

Comment:
Shouldn't be `db.io` ?

Reply:
it's db.innodb.io.size{db.io.direction="received"}